code/src/Styles.js

```diff
@@ -0,0 +1,123 @@
+import styled from 'styled-components';
+
+export const Main = styled.main`
+  display: flex;
+  justify-content: center;
+  background: rgb(255,250,250, 0.5);
+  height: 100vw;
```

Comment:
I wonder what the 100vw here in height does? I tried changing it in the development tool but doesn't seem like the it was taken.